PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -33,8 +31,10 @@ enum ValuesEvSel : int {
   All = 0,
   Cent,
   CentSel8,
-  CentSel8PosZ,
-  CentSel8PosZTFBorder,
+  CentSel8TFBorder,
+  CentSel8TFBorderPvContr,
+  CentSel8TFBorderPvContrChi2,
+  CentSel8TFBorderPvContrChi2PosZ,
```

Comment:
This enum can be removed if we use `EventRejection` for the histogram filling as well (in the correct order).

Reply:
Done